src/group/api/groups.js

```diff
@@ -83,6 +83,8 @@ export function convert (val) {
     return val.map(convert)
   }
   else {
+    val.information = val.description
+    delete val.description
```

Comment:
Ideally, it would work with both the old backend and the new backend, but that adds some complexities, and I don't think it's worth it, as I can update the two at about the same time.

So, because of that I would just remove this bit of logic here.

(and with a backend that _only_ supplies the `information` field, this code would override that with the value of `description`, which would be `undefined`)